server/CVCMIServer.cpp

```diff
@@ -1,679 +1,962 @@
-/*

```

Comment:
Whole file changed. Is something wrong with line endings?

Reply:
I actually changed 95% of code in it so I seen to reason to preserve any history. Usually I only make sure line endings are fine in new files, but I think it's make sense to do this and formatting when I actually checked all that code myself.

This also apply to CClient.